spotify.py

```diff
@@ -0,0 +1,192 @@
+from datetime import datetime, timedelta
+import time
+from spotify_secrets import DEVICE_ID, CLIENT_ID, CLIENT_SECRET, REFRESH_TOKEN
+import requests
+
+# Methods (available to use):
+# RETURN VALUES
+# returns {ERROR or CONNECTION_ERROR} dependong on error
+# nothing if successful
+
+# starts/resumes the playback
+# start_playback()
+
+# pauses the playback
+# pause_playback()
+
+# skips to the next song and plays it
+# skip_playback()
+
+# goes back to a previous song and plays it
+# previous_playback()
+
+class _Helper:
+    # Token info storage
+    access_token = None
+    token_expiration = None
+
+    # Return header info for API calls
+    def get_headers():
```

Comment:
Needs parameter self.

Reply:
I'm not super familiar with python so can you explain why this specific function needs self as a parameter?

Reply:
It has to do with how python uses Classes. It essentially needs to know to which instance of the class is calling the method. For more information refer to this [link](https://www.geeksforgeeks.org/self-in-python-class/)